lib/honeybadger/fingerprint_adapter.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Honeybadger.FingerprintAdapter do
+  @moduledoc """
+  The callbacks required to implement the FingerprintAdapter behaviour
+  """
+
+  @callback parse(Notice.noticeable()) :: String.t()
```

Comment:
It would be nice to document the callback itself as well. Possibly with some examples of returning a string based on the notice itself, examples of what a fingerprint should look like, etc.